exercises/binary_converter.cpp

```diff
@@ -1,7 +1,32 @@
-/*
-  Write a program that given a number as input convert it in binary.
+#include <iostream>
+#include <vector>
 
-  Output:
-  Insert first number: 8
-  The binary number is: 1000
-*/
+int main() {
+    int num;
+    std::cout << "Insert first number: ";
+    std::cin >> num;
+
+    // Gestione del caso 0
+    if (num == 0) {
+        std::cout << "The binary number is: 0\n";
+        return 0;
+    }
+
+    // Vector per memorizzare i bit
+    std::vector<int> binary;
+
+    // Conversione in binario
+    while (num > 0) {
+        binary.push_back(num % 2);
+        num /= 2;
+    }
+
+    // Stampa in ordine inverso
+    std::cout << "The binary number is: ";
+    for (int i = binary.size() - 1; i >= 0; --i) {
+        std::cout << binary[i];
+    }
+    std::cout << std::endl;
+
+    return 0;
```

Comment:
Il `return 0` è poco esplicativo di per sè.
Potresti fare un:
`#define EXIT_SUCCESS 0` e inserire alla fine del main `return EXIT_SUCCESS;` in modo tale da incrementare la leggibilità del codice.

```suggestion
    return EXIT_SUCCESS;
```